File: nelio-alves-algoritmo-e-logica-de-programacao/java/src/exercicios/Glicose.java
package exercicios;

import java.util.Locale;
import java.util.Scanner;

public class Glicose {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double medidaGlicose;

	    System.out.print("Digite a medida da Glicose: ");
		medidaGlicose = sc.nextDouble();

		if (medidaGlicose <= 100) {
			System.out.print("Classificacao: normal");
		}
		else if (medidaGlicose <= 140) {
			System.out.print("Classificacao: elevado");
		} else {
			System.out.print("Classificacao: diabetes");
		}

		sc.close();
	}
}

File: nelio-alves/algoritmo-e-logica-de-programacao/python/exercicios/glicose.py
medidaGlicose: float

medidaGlicose = float(input("Digite a medida da glicose: "))

if medidaGlicose <= 100:
    print("Classificacao: normal")
elif medidaGlicose <= 140:
    print("Classificacao: elevado")
else:
    print("Classificacao: diabetes")
